docs/source/recipes/running-elyra-in-air-gapped-environment.md

```diff
@@ -62,5 +62,8 @@ During pipeline execution in the Kubeflow Pipelines or Apache Airflow environmen
         - `ELYRA_REQUIREMENTS_URL` (URL of `.../etc/generic/requirements-elyra.txt`)
     - For Apache Airflow:
         - `ELYRA_BOOTSTRAP_SCRIPT_URL` (URL of `.../elyra/airflow/bootstrapper.py`)
-        - `ELYRA_REQUIREMENTS_URL` (URL of `.../etc/generic/requirements-elyra.txt`)     
+        - `ELYRA_REQUIREMENTS_URL` (URL of `.../etc/generic/requirements-elyra.txt`)
+    
+  Alternatively, for Kubeflow Pipeline users can store the bootstrapper.py, pip.conf and requirements-elyra.txt files in the runtime container itself, so it could used without having to set the above variables.The Path of these files needs to be configured, so elyra execution can pick it up.
```

Comment:
```suggestion
  Alternatively, for Kubeflow Pipeline users can store the bootstrapper.py, pip.conf and requirements-elyra.txt files in the runtime container itself, so it could be used without having to set the above variables. The path to these files needs to be configured, so Elyra execution can pick it up.
```

Reply:
thank you